resources/Makefile.j2

```diff
@@ -198,4 +195,10 @@ pheval:
 	$(MAKE) pheval-run
 	$(MAKE) pheval-report
 
+.PHONY: all
+all:
+	$(MAKE) setup
```

Comment:
Does this goal exist without `custom.Makefile`? 

Remember the makefile needs to work even with an empty `custom.Makefile`!